js/views/callinfoview.js

```diff
@@ -167,10 +184,11 @@
 				inputPlaceholder: t('spreed', 'Name'),
 				buttonTitle: t('spreed', 'Rename')
 			});
+
 		},
 
 		renderWhenInactive: function() {
-			if (this.ui.passwordInput.length === 0 || !this.passwordInputIsShown || this.ui.passwordInput.val() === '') {
```

Comment:
This is required for firefox, when you have one password saved in your browser.
Otherwise the call info will never update, because the password is always non-empty

Reply:
With the auto-fill?

Reply:
Damn firefox!!

Reply:
https://github.com/nextcloud/spreed/pull/717/files :(